core/src/main/kotlin/Kord.kt

```diff
@@ -402,7 +403,26 @@ public class Kord(
             }
             return KordRestOnlyBuilder(token).apply(builder).build()
         }
+
+        /**
+         * Builds a [Kord] instance configured by the [builder].
+         *
+         * The instance only allows for configuration of REST related APIs,
+         * interacting with the [gateway][Kord.gateway] or its [events][Kord.events] will result in no-ops.
+         *
+         * Similarly, [cache][Kord.cache] related functionality has been disabled and
+         * replaced with a no-op implementation.
+         */
+        @KordExperimental
+        public inline fun proxy(applicationId: Snowflake, builder: KordProxyBuilder.() -> Unit = {}): Kord {
```

Comment:
I think this still needs some better documentation to explain it's purpose. Do you have some suggestions?